docs/pages/blog/mui-core-v5.md

```diff
@@ -531,8 +531,23 @@ It's new, link core and x roadmaps.
 
 ### Core
 
-- Unstyled
-- Joy
+#### Unstyled components and hooks
+
+We still have a lot of work to do to have a great set of unstyled components.
```

Comment:
```suggestion
We still have a lot of work to do to have a full set of unstyled components.
```

Unless we want to remain imprecise to avoid complains if we don't create the unstyled version of some components.